graceoncampus/goc-web-v3
# Add header and footer to rides pages

Testing Steps
------------------------------------------------------------------------
1. yarn && yarn build && yarn start
2. navigate to localhost:3000
3. ensure that header + footer render for all rides endpoints

Resources
------------------------------------------------------------------------
- Figma Mock: https://www.figma.com/file/lW8XjLzvK1fRqzetVqOs7q/GOC?node-id=345%3A2127
